config/software/chef.rb

```diff
@@ -49,7 +49,7 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  excluded_groups = %w{server docgen maintenance travis}
+  excluded_groups = %w{server docgen maintenance pry travis}
```

Comment:
might be worth noting somewhere that pry-byebug has build issues with ruby 2.1.whatever so someone who wants to change this in the future knows what to look out for.